src/flow/src/server.rs

```diff
@@ -303,3 +323,116 @@ impl FlownodeBuilder {
         Ok(man)
     }
 }
+
+pub struct RemoteFrondendInvoker {
```

Comment:
typo, and also this is `Remote` here is not quite accurate.

Reply:
+1 Using `FrontendInvoker` is OK

Reply:
I think we don't need `FrontendInvoke` anymore; using just `inserter` and `deleter` is sufficient. Another layer of abstraction isn't necessary.

Reply:
Or `FrontendInvoker` can be a struct:

```
struct FrontendInvoker {
    inserter: InserterRef,
    deleter: DeleterRef,
}
```